util/src/androidInstrumentedTest/kotlin/com/wire/kalium/util/FileTestHelper.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.util
+
+import java.io.File
+import kotlin.io.path.Path
+import kotlin.io.path.absolutePathString
+import kotlin.io.path.createTempDirectory
+
+actual object FileTestHelper {
+    actual fun createRandomDirectory(): String {
+        return createTempDirectory("testDirectory").absolutePathString()
+    }
+
+    actual fun createRandomFileAt(path: String) {
+        kotlin.io.path.createTempFile(directory = Path(path), prefix = "testFile")
```

Comment:
why not import this fun, so you'll have only `createTempFile(directory = Path(path), prefix = "testFile")` here?